drivers/overlay/overlay.go

```diff
@@ -431,6 +439,25 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return "", err
 	}
 
+	newlowers := ""
+	for _, lower := range strings.Split(string(lowers), ":") {
+		newpath := path.Join(d.home, lower)
+		if _, err := os.Stat(newpath); err != nil {
+			for _, p := range d.AdditionalImageStores() {
+				lower := path.Join(p, d.name, lower)
+				_, err = os.Stat(lower)
+				if err == nil {
+					break
+				}
+			}
```

Comment:
Yes, but that's happening outside of the scope of the variable with that name that's being defined on line 447, so unless I'm mistaken, the value is going to be the one assigned on line 443.